crowbar_framework/app/models/node_object.rb

```diff
@@ -1249,6 +1249,14 @@ def poweroff
     bmc_cmd("power off")
   end
 
+  def service(service_name, action)
+    case service_name
+    when :chef
+      init_cmd = ChefObject.get_service_init_cmd(@node["platform"], @node["platform_version"], "chef-client", action)
+    end
+    ssh_cmd(init_cmd) unless init_cmd.nil?
```

Comment:
If `init_cmd` is `nil` then surely there should be at least a warning, if not an error?